nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-controller-service.js

```diff
@@ -2624,6 +2788,26 @@
             });
         },
 
+        /**
+         * Prompts the user before attempting to move the specified controller service.
+         *
+         * @param {jQuery} serviceTable
+         * @param {object} controllerServiceEntity
+         */
+        promptToMoveController: function (serviceTable, controllerServiceEntity) {
+            // prompt for move
+            nfDialog.showYesNoDialog({
+                headerText: 'Move Controller Service',
+                dialogContent: 'Move controller service \'' + nfCommon.escapeHtml(controllerServiceEntity.component.name) + '\' '
+                                + 'to \'' + nfCommon.escapeHtml($('#move-controller-service-scope').combo('getSelectedOption').text) + '\'?'
+                                + '<br /><br /><i class="invalid"></i> &nbsp'
+                                + 'Any reference to this controller service by processors outside the new scope will be removed.',
+                yesHandler: function () {
+                    moveToProcessGroup(serviceTable, controllerServiceEntity, $('#move-controller-service-scope').combo('getSelectedOption').value);
+                }
+            });
+        },
```

Comment:
I don't think we need a prompt, either, after ensuring that references cannot become invalid. And because the service doesn't require being disabled, I think the option should always be shown.

Reply:
Removed prompt. Made it to where the controller service can be moved while enabled.